### PR TITLE
feat(PROTECT-1782): Change the wording around login to recover app

### PR DESCRIPTION
**Pull request by andreishtamburg**
### 📝 Description

Changes in wording on the recover app login

### ❓ Context

- **Impacted projects**: `LLM`, `LLD`
- **Linked resource(s)**: `PROTECT-1782`, `PROTECT-2171`

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by vcluzeau-ledger — APPROVED** [review]

File: apps/ledger-live-mobile/src/locales/en/common.json [hunk 1]
```diff
@@ -6265,10 +6265,10 @@
       "status": {
         "new": {
           "title": "New",
-          "desc": "Recover access to your wallet without your Secret Recovery Phrase.",
+          "desc": "Stop worrying about getting locked out of your wallet.",
           "actions": {
             "learnMore": "How does it benefit me?",
-            "alreadySubscribed": "Already logged in? <LinkAccount>View my subscription</LinkAccount>"
+            "alreadySubscribed": "I already have a log in <LinkAccount>View my subscription</LinkAccount>"
```
Comment (sshmaxime) [inline apps/ledger-live-mobile/src/locales/en/common.json:6271]: Why not `I already have an account`, otherwise it's good to go imo.

Comment (sshmaxime) [inline apps/ledger-live-mobile/src/locales/en/common.json:6271]: Why not `I already have an account` ?

**Review by sshmaxime — APPROVED** [review]
Lgtm, although when you open a PR and are filling it this is how you would want to write the context section:

- **Impacted projects**: `LLM` or `LLD` (or other if necessary but usually it would be those two)
- **Linked resource(s)**: [LIVE-8173] (You have to remove the bracket like that `` around)

File: apps/ledger-live-mobile/src/locales/en/common.json (hunk 1 above)
Comment (andreishtamburg) [reply apps/ledger-live-mobile/src/locales/en/common.json:6271]: Don't get me started on this :) 
Driven by communication around whole `Recover` shenanigans

Comment (sshmaxime) [reply apps/ledger-live-mobile/src/locales/en/common.json:6271]: Ups 🤣

**Pull request merged**